frontend/src/features/mapSideBar/SidebarContents/subforms/ParcelIdentificationForm.tsx

```diff
@@ -77,13 +78,30 @@ export const ParcelIdentificationForm: React.FC<IIdentificationProps> = ({
 
   const agencies = (props.agencies ?? []).map(c => mapSelectOptionWithParent(c, props.agencies));
   const formikProps = useFormikContext<ISteppedFormValues<IParcel>>();
-  const agency = getIn(formikProps.values, withNameSpace(nameSpace, 'agencyId'));
   const { lookupCodes } = useCodeLookups();
   const { propertyTypeId, latitude, longitude } = getIn(formikProps.values, nameSpace);
   const projectNumbers = getIn(formikProps.values, 'data.projectNumbers');
   const agencyId = getIn(formikProps.values, `data.agencyId`);
   const [privateProject, setPrivateProject] = useState(false);
 
+  const keycloak = useKeycloakWrapper();
```

Comment:
My PR has a hook that allows to get the user's agencies, [](https://github.com/bcgov/PIMS/pull/1038)

Reply:
oooh thank you!